packages/loader/src/resource-deserialize/resources/prefab/PrefabDesign.ts

```diff
@@ -63,3 +86,16 @@ export type IClassObject = {
 export type IBasicType = string | number | boolean | null | undefined | IAssetRef | IClassObject | IMethodParams;
 
 export type IAssetRef = { key?: string; refId: string };
+
+export interface IPrefabMaterial {
```

Comment:
This should named `IMaterialSchema` or `IMaterialDefine`

Reply:
+1